config/locales/ar/world_location_news.yml

```diff
@@ -1,5 +1,12 @@
----
```

Comment:
what's up with these lines / lines getting removed? Can we just take them out of the original PR if we don't need them?

Reply:
This is an artefact of me branching off an old version of another branch.  I've rebased, so the lines will remain now (they are put there by `rake translation:normalize`).